schematic/models/validate_attribute.py

```diff
@@ -26,6 +26,20 @@
 logger = logging.getLogger(__name__)
 
 class GenerateError:
+    def generate_schema_error(row_num: str, attribute_name: str, error_msg: str)-> List[str]:
+        '''
+        Purpose: Process error messages generated from schema
+        Input:
+            - row_num: the row the error occurred on.
+            - attribute_name: the attribute the error occurred on.
+            - error_msg: Error message
+        '''
+        arg_error_string = (
```

Comment:
Can you reformat the variable names and the output to follow the format of the other error methods?

Reply:
Could you review it again? I try to make it looks like the other functions in terms of formatting. If you want me to rename other variables, please let me know.